src/makeRobot.js

```diff
@@ -38,7 +40,71 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+
+      this.coords.y += step;
```

Comment:
```suggestion
      if (step > 0) {
        this.coords.y += step;
      }
```
You will `return this` anyway. Just change the coord only if it's positive value
Fix all cases